Week-2/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Week 2 Assignment 1 - Loren Ipsum</title>
+        <link rel="stylesheet" type="text/css" href="./csssheet.css">
+    </head>
+    <body>
+        <div id="mainhead">
+            <h1>Lorem Ipsum</h1>
+    </div>
+        <div class="content">
+        <div class="contentL">
+        <div>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla fermentum egestas metus, sed posuere purus faucibus at. Suspendisse lacinia blandit lorem, ut condimentum libero dapibus sit amet. Praesent eu semper erat, vel bibendum massa. Aenean pellentesque lacus eu vulputate sollicitudin. Donec id dolor vehicula, elementum orci at, convallis metus. Integer non eros non tortor accumsan dignissim vel eu urna. Praesent bibendum, lectus vitae consequat ornare, lorem purus aliquet est, eget egestas urna arcu in diam. Mauris laoreet tempor nibh eget congue. Vestibulum vitae tincidunt lacus. Morbi quis maximus justo. Vestibulum sit amet cursus quam. Ut vel mattis orci. Nunc sit amet fermentum nulla, sed vestibulum mi. Duis varius ac est quis luctus. Curabitur in risus eu quam commodo dignissim. Morbi sed ultrices eros, eget aliquam nisi.</p>
+        </div>        
+        <div>
+            <p>Proin mollis nulla malesuada, tincidunt purus non, scelerisque nisl. Suspendisse ligula urna, porttitor vel felis sed, ullamcorper fermentum quam. Donec maximus augue ac libero rhoncus consequat. Maecenas eget hendrerit odio. Donec ut convallis nisi, vel lobortis tellus. Integer ut nisi ut sapien suscipit tempor in vitae sem. Pellentesque in ultricies libero. Aenean cursus mi efficitur quam placerat viverra. Sed ac mollis quam. Praesent a orci convallis, aliquet nisl tincidunt, scelerisque libero. Integer non urna sed lectus tempus lobortis non a mauris. Curabitur in ligula tempus lorem porta aliquam. Vestibulum at tristique metus. Praesent non dui vulputate, scelerisque justo eget, faucibus tellus.</p>
+        </div>
+        <div class="image">
+            <img src="https://cdn.shopify.com/s/files/1/0641/4866/8630/articles/img.png?v=1649204872"/>
+        </div>
+    </div>
+    <div class="contentR">
+        <div>
+            <p>Suspendisse porttitor tincidunt fermentum. Sed rutrum felis nisi, eget semper sapien ornare quis. Duis non quam ac purus tristique commodo. Vivamus auctor scelerisque dui, non ornare purus mattis eget. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin convallis porta nulla at sodales. Donec eget dignissim nisi. Nam fermentum at ligula in convallis.</p>
+        </div>
```

Comment:
@rajeshvlega9 The text and image seems not responsive. Can you look into it ?

<img width="397" alt="Screenshot 2022-07-30 at 11 10 21 AM" src="https://user-images.githubusercontent.com/9133593/181876556-6dd3b7da-d79b-4489-8319-4d16d6e3a733.png">

Reply:
This assignment i have just completed now. i believe now its responsive, please find the screenshot below:
Normal screen ![image](https://user-images.githubusercontent.com/41244432/181902539-f175c129-9f7e-46ae-85a1-e3dcc73a5ff2.png)

with smaller screen resolution
![image](https://user-images.githubusercontent.com/41244432/181902674-dae46bbd-9f11-440c-a65c-a8a86ffba514.png)